tasks/main.yml

```diff
@@ -80,24 +80,3 @@
   when: postgresql_backup_enabled
   tags:
     - backup
-
-- include_role:
-    name: onaio.ssl-certificate
-  vars:
```

Comment:
Should this move to dependencies?

Reply:
Not really. The reason these were moved to the tasks/main.yml file is because they can only run successfully after the PosgreSQL system user has been set up.